tests/integration/Owncloud/OcisPhpSdk/ShareCreatedModifyTest.php

```diff
@@ -76,17 +77,19 @@ public function testDeleteIndividualShare(): void
     public function testDeleteGroupShare(): void
     {
         $philosophyHatersGroup =  $this->ocis->createGroup(
-            'philosophyhaters',
+            'philosophy-haters',
```

Comment:
```suggestion
            'philosophyhaters',
```

We removed these `-` yesterday.